MachObfuscator/Options/Options.swift

```diff
@@ -28,26 +36,66 @@ extension Options {
         var debug = false
         var machOViewDoom = false
         var methTypeObfuscation = false
+        var swiftReflectionObfuscation = false
+        var obfuscableFilesFilter = ObfuscableFilesFilter.defaultObfuscableFilesFilter()
         var manglerKey = SymbolManglers.defaultManglerKey
-        while case let option = getopt(argc, unsafeArgv, "qvdhtDm:"), option != -1 {
-            let char = UnicodeScalar(CUnsignedChar(option))
-            switch char {
-            case "q":
+        
+        struct OptLongChars {
+            static let unknownOption = Int32(Character("?").asciiValue!)
+            static let help = Int32(Character("h").asciiValue!)
+            static let verbose = Int32(Character("v").asciiValue!)
+            static let quiet = Int32(Character("q").asciiValue!)
+            static let debug = Int32(Character("d").asciiValue!)
+            static let methTypeObfuscation = Int32(Character("t").asciiValue!)
+            static let machOViewDoom = Int32(Character("D").asciiValue!)
+            static let manglerKey = Int32(Character("m").asciiValue!)
+        }
+        enum OptLongCases: Int32 {
+            case OPT_FIRST = 256
+            case swiftReflection
+            case noFramework
+            case noFrameworks
+        };
+        
+        let longopts: [option] = [
+            option(name: Options.newCCharPtrFromStaticString("help"),      has_arg: no_argument,       flag: nil, val: OptLongChars.help),
+            option(name: Options.newCCharPtrFromStaticString("verbose"),   has_arg: no_argument,       flag: nil, val: OptLongChars.verbose),
+            option(name: Options.newCCharPtrFromStaticString("methtype"),      has_arg: no_argument, flag: nil, val: OptLongChars.methTypeObfuscation),
+            option(name: Options.newCCharPtrFromStaticString("machoview-doom"),      has_arg: no_argument, flag: nil, val: OptLongChars.machOViewDoom),
+            option(name: Options.newCCharPtrFromStaticString("swift-reflection"),      has_arg: no_argument, flag: nil, val: OptLongCases.swiftReflection.rawValue),
+            option(name: Options.newCCharPtrFromStaticString("no-framework"),      has_arg: required_argument, flag: nil, val: OptLongCases.noFramework.rawValue),
```

Comment:
What do you think about changing option name to `skip-framework`?

Reply:
done